Examples/Swift/ViewController.swift

```diff
@@ -266,6 +268,14 @@ extension ViewController: NavigationViewControllerDelegate {
         
         navigationViewController.present(confirmationController, animated: true, completion: nil)
     }
+    
+    private func showEndOfRouteFeedback(for waypoint: Waypoint) {
```

Comment:
This should probably me moved out of the example and into `NavigationViewController.swift`

Reply:
Right now, our concept of route ratings only applies to the entire route rather than individual legs, so let's only show a star rating for the final waypoint rather than intermediate waypoints. This should be configurable with a simple bool like we do for the feedback button currently.

The overall `arrival` UI could have a public delegate method or invoke function, as you've suggested @bsudekum.